pyslim/methods.py

```diff
@@ -84,3 +86,93 @@ def recapitate(ts,
     return SlimTreeSequence(recap, reference_sequence=ts.reference_sequence)
 
 
+def convert_alleles(ts, generate=True, seed=None):
```

Comment:
I would have thought it would be clearer to have two functions ``convert_alleles`` which expects a SLiM nucleotide model and ``generate_alleles`` which will generate random alleles. It took a few goes at reading to docstring to get the current semantics (why would you not use the SLiM nucleotides, if they exist already?)

Reply:
Hm, that's a nice idea. Right now it's using nucleotides when they exist and generating them if they don't, so it works with mixed nuc- and non-nuc mutations. But there's no need to make it so complex.